### PR TITLE
Exclusion Strategy

### LINKED ISSUE #12: Add strategy to exclude certain entities
Sometimes it might be useful to exclude certain entities from generating a fixture file, so some sort of strategy to do this might be a good idea.

### PR DESCRIPTION
**Pull request by miguelgf**
Posibility to define a strategy to exclude some entities to be dumped. Includes an implementation of this strategy to define an array of class names or FQN to be excluded.

Closes #12

### CODE REVIEW

**Comment by infinit89** [comment]
ping @Spea

File: src/Sp/FixtureDumper/ExclusionStrategy/ArrayExclusionStrategy.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    /**
+     * {@inheritdoc}
+     */
+    public function shouldSkipClass(ClassMetadata $metadata)
+    {
+        return in_array(ClassUtils::getClassName($metadata->getName()), $this->skipClassesNames) ||
+               in_array($metadata->getName(), $this->skipClassesNames);
+    }
+}
```
Comment (Spea) [inline src/Sp/FixtureDumper/ExclusionStrategy/ArrayExclusionStrategy.php]: Missing the new line here aswell

**Review by Spea — COMMENTED** [review]
Only one more change, will merge it then :)

**Comment by Spea** [comment]
Thx for the PR. Will create a new release sometime today.

**Pull request merged**